docs/en/stack/ml/df-analytics/flightdata-regression.asciidoc

```diff
@@ -0,0 +1,408 @@
+[role="xpack"]
+[testenv="platinum"]
+[[flightdata-regression]]
+=== Predicting delays with {reganalysis} by using Kibana sample flight data
+
+Let's try to predict flight delays by using the 
+{kibana-ref}/add-sample-data.html[sample flight data]. We use this data to learn 
+a {regression} model that allows us to predict flight delays based on 
+information such as weather and location of the destination and origin, flight 
+distance and carrier. We'd like to predict the number of minutes delayed for 
+each flight. As it is a continuous numeric variable, we'll use {reganalysis} to 
```

Comment:
> ... we'll use {reganalysis}...

Since we're only deciding to use regression analysis at this point in the procedure, does this mean it was too soon to mention a "regression model" in an earlier sentence?

Reply:
I think at this point we explain why we use regression analysis instead of anything else, in this regard, it is not the moment of decision.